File: app/src/main/java/com/example/helloworld/QuestionLibrary.java
package com.example.helloworld;

public class QuestionLibrary {
    private String[] Questions = {
            "Jakie objaw nie towarzyszy podczas zakażenia koronawirusem ?",
            "Jaka grupa wiekowa ludzi jest narażona najbardziej na śmierć ?",
            "W jakim kraju zaczeła sie Pandemia ?",
            "Czy noszenie maseczek ma sens ?",
            "W jakim województwie w Polsce jest najwięcej zarażonych",
            "Który kraj do tej pory ma najwięcej zgonów w ciągu doby?",
            "Koniec QUIZU twój wynik to"

    };


    private String[][] Choices = {
            {"Kaszel", "Gorączka", "Ból brzucha"},
            {"10-30", "30-50", "50+"},
            {"Polska", "Chiny", "Włochy"},
            {"Tak", "Nie", "Zależy czy jesteś chory czy nie"},
            {"Pomorskie", "Małopolskie", "Mazowieckie"},
            {"Włochy", "USA", "Hiszpania"},
            {"Włochy", "USA", "Hiszpania"}
    };



    private String[] CorrectAnswers = {"Ból brzucha", "50+", "Chiny", "Zależy czy jesteś chory czy nie", "Mazowieckie", "Włochy", "Włochy"};




    public String getQuestion(int a) {
        String question = Questions[a];
        return question;
    }


    public String getChoice1(int a) {
        String choice0 = Choices[a][0];
        return choice0;
    }


    public String getChoice2(int a) {
        String choice1 = Choices[a][1];
        return choice1;
    }

    public String getChoice3(int a) {
        String choice2 = Choices[a][2];
        return choice2;
    }

    public String getCorrectAnswer(int a) {
        String answer = CorrectAnswers[a];
        return answer;
    }

}

File: app/src/main/java/com/example/helloworld/QuestionLibrary.kt
package com.example.helloworld

class QuestionLibrary {
    private val Questions = arrayOf(
            "Jakie objaw nie towarzyszy podczas zakażenia koronawirusem ?",
            "Jaka grupa wiekowa ludzi jest narażona najbardziej na śmierć ?",
            "W jakim kraju zaczeła sie Pandemia ?",
            "Czy noszenie maseczek ma sens ?",
            "W jakim województwie w Polsce jest najwięcej zarażonych",
            "Który kraj do tej pory ma najwięcej zgonów w ciągu doby?")
    private val Choices = arrayOf(arrayOf("Kaszel", "Gorączka", "Ból brzucha"), arrayOf("10-30", "30-50", "50+"), arrayOf("Polska", "Chiny", "Włochy"), arrayOf("Tak", "Nie", "Zależy czy jesteś chory czy nie"), arrayOf("Pomorskie", "Małopolskie", "Mazowieckie"), arrayOf("Włochy", "USA", "Hiszpania"))
    private val CorrectAnswers = arrayOf("Ból brzucha", "50+", "Chiny", "Zależy czy jesteś chory czy nie", "Mazowieckie", "Włochy")
    fun getQuestion(a: Int): String {
        return Questions[a]
    }

    fun getChoice1(a: Int): String {
        return Choices[a][0]
    }

    fun getChoice2(a: Int): String {
        return Choices[a][1]
    }

    fun getChoice3(a: Int): String {
        return Choices[a][2]
    }

    fun getCorrectAnswer(a: Int): String {
        return CorrectAnswers[a]
    }
}